libs/wasm-transform/src/transform.rs

```diff
@@ -62,12 +271,13 @@ pub fn truncate(req: &Request) -> Result<String, TransformError> {
             } else {
                 length_of_field - truncate_options.length.clone()
             }
-        },
+        }
         TruncateType::Percent => {
-            let my_usize_reference = 100.0 - &truncate_options.length.value_as::<f64>().unwrap_or(0.0);
+            let my_usize_reference =
+                100.0 - &truncate_options.length.value_as::<f64>().unwrap_or(0.0);
```

Comment:
That is a lot of chars for `100-float64(truncateOptionsLength)` 😐

Reply:
Yay rust! 😆